software/digiread.yml

```
@@ -0,0 +1,12 @@
+name: Digiread
+website_url: https://ladigitale.dev/digiread/
+description: Clean up online pages and articles using Mozilla's Readability (documentation in French).
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - PHP
+tags:
+  - Feed Readers
```

```suggestion
  - Miscellaneous
```

It's not actually a feed reader